src/applications/disability-benefits/all-claims/tests/containers/Form526EZApp.unit.spec.jsx

```diff
@@ -147,60 +149,142 @@ describe('Form 526EZ Entry Page', () => {
     tree.unmount();
   });
 
-  // Logged in & verified, but missing ID
-  it('should render Missing ID page', () => {
-    sessionStorage.setItem(WIZARD_STATUS, WIZARD_STATUS_COMPLETE);
-    const tree = testPage({
-      currentlyLoggedIn: true,
-      verified: true,
-      services: [],
+  describe('Missing user identifiers', () => {
+    // claims.form526RequiredIdentifierPresence is included in the profile when the form_526_required_identifiers_in_user_object feature flag is enabled on the back end
```

Comment:
Because the following test pertains to a claims object **without** `form526RequiredIdentifierPresence`, I would suggest modifying the wording to the effect of "...is **not** included..." and "...is **not** enabled..."